docs/demo_examples/plot_3D_cyl.rst

```diff
@@ -0,0 +1,74 @@
+.. note::
```

Comment:
is this auto-generated? if so, then we don't need to include it in the repository